src/lib/namron.ts

```diff
@@ -66,6 +108,68 @@ export const fromZigbee = {
             return result;
         },
     } satisfies Fz.Converter<"hvacThermostat", undefined, ["attributeReport", "readResponse"]>,
+    namron_relay_ntc_water_sensor: {
```

Comment:
This can be refactored to modernExtend ([example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/5d8679fd1688273ac011f4df60c0c67870951298/src/devices/custom_devices_diy.ts#L882)), also add a custom cluster definition ([example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/5d8679fd1688273ac011f4df60c0c67870951298/src/devices/sonoff.ts#L213))